tests/storage/cdi_clone/test_clone.py

```diff
@@ -193,48 +191,50 @@ def test_successful_vm_from_cloned_dv_windows(
         )
 
 
-@pytest.mark.xfail(
-    reason=f"{QUARANTINED}: consistently failing test. Timeout into ssh connection to the vm; CNV-66721",
-    run=False,
-)
 @pytest.mark.sno
-@pytest.mark.parametrize(
-    "data_volume_multi_storage_scope_function",
-    [
-        pytest.param(
-            {
-                "dv_name": "dv-source",
-                "image": FEDORA_LATEST.get("image_path"),
-                "dv_size": Images.Fedora.DEFAULT_DV_SIZE,
-            },
-            marks=(pytest.mark.polarion("CNV-4035")),
-        )
-    ],
-    indirect=True,
-)
 @pytest.mark.s390x
+@pytest.mark.polarion("CNV-4035")
 def test_disk_image_after_clone(
-    skip_block_volumemode_scope_function,
     unprivileged_client,
     namespace,
-    data_volume_multi_storage_scope_function,
+    storage_class_name_scope_module,
+    fedora_data_source_scope_module,
     cluster_csi_drivers_names,
 ):
-    storage_class = data_volume_multi_storage_scope_function.storage_class
-    with create_dv(
-        source="pvc",
-        dv_name="dv-cnv-4035",
+    source_dict = fedora_data_source_scope_module.source.instance.to_dict()
+    source_spec_dict = source_dict["spec"]
+    size = source_spec_dict.get("resources", {}).get("requests", {}).get("storage") or source_dict.get(
+        "status", {}
+    ).get("restoreSize")
+
+    with DataVolume(
+        name="dv-cnv-4035",
         namespace=namespace.name,
-        size=data_volume_multi_storage_scope_function.size,
-        source_pvc=data_volume_multi_storage_scope_function.name,
         client=unprivileged_client,
-        storage_class=storage_class,
+        size=size,
+        api_name="storage",
+        storage_class=storage_class_name_scope_module,
+        source_ref={
+            "kind": fedora_data_source_scope_module.kind,
+            "name": fedora_data_source_scope_module.name,
+            "namespace": fedora_data_source_scope_module.namespace,
+        },
     ) as cdv:
-        cdv.wait_for_dv_success()
-        create_vm_and_verify_image_permission(dv=cdv)
+        cdv.wait(timeout=TIMEOUT_10MIN, wait_for_exists_only=True)
```

Comment:
No need for such a long timeout 

```suggestion
        cdv.wait(wait_for_exists_only=True)
```